commerce_coordinator/apps/rollout/utils.py

```diff
@@ -30,15 +31,18 @@ def get_order_return_info_return_items(order: CTOrder) -> List[CTReturnItem]:
     return return_info_items
 
 
-def is_commercetools_line_item_already_refunded(order: CTOrder, order_line_item_id: str) -> bool:
+def is_commercetools_line_item_already_refunded(order: CTOrder, order_line_item_ids: List[str]) -> bool:
     """
     Determine if a return already exists for the Commercetools line item
     to prevent duplicate refunds/returns.
     """
 
     return_info_return_items = get_order_return_info_return_items(order)
 
-    return len(list(filter(lambda item: item.line_item_id == order_line_item_id, return_info_return_items))) >= 1
+    return len(list(filter(
+        lambda item: item.line_item_id in order_line_item_ids and item.payment_state == ReturnPaymentState.REFUNDED,
+        return_info_return_items
+    ))) >= 1
```

Comment:
this will check if any of the item is refunded, it will return true, do we want this? i think we would want to return other line items if anyone is already refunded.

Reply:
yes, i think we should refund line items that are not already refunded